network/network.go

```diff
@@ -25,6 +25,8 @@ const (
 const (
 	// ipv6 modes
 	IPV6Nat = "ipv6nat"
+	// dual stack mode
+	DualStack = "dualStackOverlay"
```

Comment:
i see you are using this only in network_windows.go. can you move it there? also update name of variable to "dualStackOverlay"

Reply:
variable name has changed, it's also used in network.go to add routes